File: java/doit_java/Chapter5/src/classpart/Student.java
// Student 클래스 (Student5와 같다)
package classpart;

public class Student {
    int studentId;      // 학번
    String studentName; // 학생 이름
    int grade;          // 학년
    String address;     // 사는 곳

    // 학생의 이름을 반환하는 메서드
    public String getStudentName() {
        return studentName;
    }

    // 학생의 이름을 부여하는 메서드
    public void setStudentName(String name) {
        studentName = name;
    }

    // 메인 함수
    public static void main(String[] args) {
        Student studentLim = new Student();  // Student 클래스 생성
        studentLim.studentName = "임채영";

        System.out.println(studentLim.studentName);
        System.out.println(studentLim.getStudentName());
    }
}

File: python/python_oop_practice/p_oop_2/chap05/classpart/student.py
class Student:
    def get_student_name(self):
        return self.student_name

    def show_studnet_info(self):
        print(self.student_name + ", " + self.address)


def main():
    student_lee = Student()
    student_lee.student_name = "이순신"
    student_lee.address = "서울"

    student_lee.show_studnet_info()

    student_kim = Student()
    student_kim.student_name = "김유신"
    student_kim.address = "경주"

    student_kim.show_studnet_info()

    print(student_lee)
    print(student_kim)


if __name__ == '__main__':
    main()
